peerprep/frontend/src/views/CollabServiceViews/CollabServiceIntegratedView.tsx

```diff
@@ -218,92 +213,96 @@ const CollaborationServiceIntegratedView: React.FC = () => {
+        <button
+          onClick={handleAssesCode}
+          className="run-btn"
+          style={{ marginBottom: '0px' }}
+        > Assess Code
+        </button>
+      </div>
 
+      <div className="code-and-chat">
         <div className="editor-container">
-            <CodeMirror
+          <CodeMirror
             ref={editorRef}
             options={{
-                mode: syntaxLang,
-                lineNumbers: true,
-                tabSize: 2,
-                indentWithTabs: true,
-                showHint: true,
-                extraKeys: {
-                    'Ctrl-Space': 'autocomplete', // Trigger autocomplete with Ctrl-Space
-                },
-                hintOptions: { completeSingle: false },
+              mode: syntaxLang,
+              lineNumbers: true,
+              tabSize: 2,
+              indentWithTabs: true,
+              showHint: true,
+              extraKeys: {
+                'Ctrl-Space': 'autocomplete', // Trigger autocomplete with Ctrl-Space
+              },
+              hintOptions: { completeSingle: false },
             }}
 
             editorDidMount={(editor) => {
-                editor.on('keyup', (cm: any, event: any) => {
-                    // Only trigger autocomplete on specific characters
-                    const triggerKeys = /^[a-zA-Z0-9_]$/; // Allow letters, numbers, and underscore
-                    if (
-                        triggerKeys.test(event.key) &&
-                        !cm.state.completionActive // Ensure that completion is not already active
-                    ) {
-                        cm.showHint({ completeSingle: false });
-                    }
-                });
+              editor.on('keyup', (cm: any, event: any) => {
+                // Only trigger autocomplete on specific characters
+                const triggerKeys = /^[a-zA-Z0-9_]$/; // Allow letters, numbers, and underscore
+                if (
+                  triggerKeys.test(event.key) &&
+                  !cm.state.completionActive // Ensure that completion is not already active
+                ) {
+                  cm.showHint({ completeSingle: false });
+                }
+              });
             }}
             onChange={() => {
-                // Let Yjs handle all updates; do not use setCode here
+              // Let Yjs handle all updates; do not use setCode here
             }}
-            />
+          />
         </div>
-      
+        {sessionId && <Chat sessionId={sessionId} />}
+      </div>
+
       <h3 style={{ textAlign: 'left', marginBottom: '5px' }}>Output</h3>
-      <div className="output-container" style={{ width: '900px', textAlign: 'left', border: '1px solid #ddd', padding: '10px', borderRadius: '5px', backgroundColor: '#f9f9f9', overflowY: 'scroll' }}>
+      <div className="output-container" style={{ width: '100%', textAlign: 'left', border: '1px solid #ddd', padding: '10px', borderRadius: '5px', backgroundColor: '#f9f9f9' }}>
```

Comment:
Add the "overflowY:" scroll here
```suggestion
      <div className="output-container" style={{ width: '100%', textAlign: 'left', border: '1px solid #ddd', padding: '10px', borderRadius: '5px', backgroundColor: '#f9f9f9', overflowY: 'scroll' }}>
```

Reply:
It is already taken care of in the CSS files :)